content/en/blog/_posts/2019-12-09-volume-snapshot-beta.md

```diff
@@ -152,7 +152,7 @@ type VolumeSnapshotContentStatus struct {
 	ReadyToUse *bool
 	RestoreSize *int64
 	Error *VolumeSnapshotError
-           SnapshotHandle *string
+  SnapshotHandle *string
```

Comment:
sorry to be picky, the indention seems to be still different, maybe tab v.s. space?